docs/data-sources/equinix_fabric_network.md

```diff
@@ -0,0 +1,99 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "equinix_fabric_network Data Source - terraform-provider-equinix"
+subcategory: ""
+description: |-
+  Fabric V4 API compatible data resource that allow user to fetch Fabric Network for a given UUID
+---
+
+# equinix_fabric_network (Data Source)
+
+Fabric V4 API compatible data resource that allow user to fetch Fabric Network for a given UUID
+
+~> **Note** Equinix Fabric v4 resources and datasources are currently in Beta. The interfaces related to `equinix_fabric_` resources and datasources may change ahead of general availability. Please, do not hesitate to report any problems that you experience by opening a new [issue](https://github.com/equinix/terraform-provider-equinix/issues/new?template=bug.md)
```

Comment:
No beta notice necessary. We're in GA for Fabric.

Reply:
This should be removed from the resource/data_source code description and then document generation will fix it.